src/storage/leveldb/wallet_index.ts

```diff
@@ -48,47 +48,29 @@ export default class LevelWalletIndex implements IKVWalletIndex {
    * Convert a number to a uint32 buffer before saving on database.
    * Internal helper method, since this logic is used in multiple places.
    *
-   * @param {'access'|'wallet'} dest Which database to use.
    * @param key The key to use when setting the value.
    * @param {number} value The value to set.
    */
-  async _setNumber(dest: 'access'|'wallet', key: string, value: number) {
+  async _setNumber(key: string, value: number) {
     if (value < 0) {
       throw new Error(`Invalid unsigned int ${value} being set on ${key}`);
     }
-    const buf = Buffer.alloc(4);
-    buf.writeUInt32BE(value);
-    switch(dest) {
-      case 'access':
-        await this.accessDB.put<string, Buffer>(key, buf, { valueEncoding: 'buffer'});
```

Comment:
You don't use this method for access db anymore?

Reply:
There is only 1 information saved as a buffer on leveldb, since we would be required to always treat for both `Buffer` and `Uint8Array` I think the best decision is to not rely on the leveldb buffer encoding and use the string encoding.
The data will be encoded in our format, which I decided on the hex value since it gives a deterministic size without the need of a encoding/decoding lib.